app/scenes/Talk/components/Pane/index.js

```diff
@@ -32,51 +34,114 @@ function Speaker({ data, onPress }) {
   return (
     <TouchableOpacity {...touchableProps}>
       <View style={styles.heroSpeaker}>
-        <Avatar source={data.avatar} />
+        <Avatar source={speaker.avatar} />
         <Text style={styles.heroSpeakerName}>
-          {data.name}
+          {speaker.name}
         </Text>
         <Text style={styles.heroSpeakerHint}>
           (tap for more)
         </Text>
       </View>
     </TouchableOpacity>
   );
-}
+};
 
 type Props = {
-  nextTalk?: ScheduleTalk,
-  nextTalkPreviewIsEngaged?: boolean,
+  nextTalk?: ScheduleTalk | null,
+  nextTalkPreviewIsEngaged: boolean,
   onHeroLayout?: (Object) => mixed,
   onPressNext?: (Object) => mixed,
-  onScroll?: (Object) => mixed,
+  onScroll?: ((Object) => mixed) | null,
   onScrollEndDrag?: () => mixed,
-  prevTalk?: ScheduleTalk,
-  prevTalkPreviewIsEngaged?: boolean,
-  showSpeakerModal: (SpeakerType | Array<SpeakerType>) => mixed,
+  prevTalk?: ScheduleTalk | null,
+  prevTalkPreviewIsEngaged: boolean,
+  showSpeakerModal: (SpeakerType) => mixed,
   visibleTalk: ScheduleTalk,
 };
```

Comment:
These props really need to be looked through better. Most of them seem necessary, but there are uses of this component that is missing most of them.